index.php

```diff
@@ -66,6 +66,7 @@
             </div>
             <table class="table table-hover">
                 <caption><i class="fa fa-rocket"></i> These ships are ready for their next Mission</caption>
+                <a href="manageShips.php" class="btn btn-md btn-primary pull-right">Manage Ships</a>
```

Comment:
I think this route should be `/manageShips.php`

Reply:
Updated route